client/packages/requisitions/src/ResponseRequisition/DetailView/IndicatorEdit/IndicatorLineEdit.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useCallback } from 'react';
+import { Footer } from './Footer';
+import {
+  BasicTextInput,
+  Box,
+  IndicatorColumnNode,
+  IndicatorValueTypeNode,
+  InputWithLabelRow,
+  NumericTextInput,
+  useNotification,
+  useTranslation,
+} from '@openmsupply-client/common';
+import {
+  IndicatorLineRowFragment,
+  IndicatorLineWithColumnsFragment,
+} from '../../api';
+import { useDraftIndicatorValue } from './hooks';
+
+interface IndicatorLineEditProps {
+  requisitionNumber: number;
+  indicatorCode?: string;
+  hasNext: boolean;
+  next: IndicatorLineRowFragment | null;
+  hasPrevious: boolean;
+  previous: IndicatorLineRowFragment | null;
+  currentLine?: IndicatorLineWithColumnsFragment | null;
+}
+
+const INPUT_WIDTH = 185;
+const LABEL_WIDTH = '150px';
+
+const InputWithLabel = ({ data }: { data: IndicatorColumnNode }) => {
+  if (!data?.value) {
+    return;
```

Comment:
![Screenshot 2024-11-27 at 12 56 38 PM](https://github.com/user-attachments/assets/6bbd817d-e0a1-424d-b1a8-9e28be19c731)

Not sure if you were just returning here so the type is definitely available but... if no value on indicator yet then I get nothing!

Reply:
Probably should just pass through a default empty string if not defined...

Reply:
If the values don't exist for the indicator then the buttons or (tab if both indicators) shouldn't show!

Reply:
Alas we have determined this was actually because I had indicator lines with no values on an existing program requisition - works fine on newly created ones! Need to find a way to handle existing requisitions 🙏